src/stores/github.ts

```diff
@@ -106,6 +108,7 @@ export const prInfoStore = create<PrInfoPropsStore>()((set) => ({
         return {
           filename: commit.filename,
           status: commit.status,
+          language: getLanguageFromFileName(commit.filename.split(".")[1]),
```

Comment:
맨 마지막이어야 해서, 1보다는 `.at(-1)`이 더 좋아요
```suggestion
          language: getLanguageFromFileName(commit.filename.split(".").at(-1),
```

Reply:
오호 이런 방법이 감사합니다. 바로 반영하도록 하겠습니다!